src/jvm/test/resources/expected_logs/concurrent_linked_deque_trace_debugger.txt

```diff
@@ -75,7 +75,7 @@ Detailed trace:
 |                                                                                                                                     |     last(): Node#1 at ConcurrentLinkedDeque.peekLast(ConcurrentLinkedDeque.java:887)                              |
 |                                                                                                                                     |       tail ➜ Node#2 at ConcurrentLinkedDeque.last(ConcurrentLinkedDeque.java:737)                                 |
 |                                                                                                                                     |       p.next ➜ Node#1 at ConcurrentLinkedDeque.last(ConcurrentLinkedDeque.java:738)                               |
-|                                                                                                                                     |       p.next ➜ null at ConcurrentLinkedDeque.last(ConcurrentLinkedDeque.java:738)                                 |
```

Comment:
Following the source code, it should be `p.next`:

```
            for (Node<E> t = tail, p = t, q;;) {
                if ((q = p.next) != null &&
                    (q = (p = q).next) != null)
```

Reply:
I think it's actually `q`, so the trace became correct:

<img width="3082" height="1188" alt="Screenshot from 2025-08-04 23-58-33" src="https://github.com/user-attachments/assets/c4d93947-11d5-4673-aa2b-d2c5089a7d58" />

Reply:
Following the code, the read should be from `p`, but
(1) the compile might generate a read from `q` -- the code is too non-trivial
(2) this code is hard to debug, so let's not spend time on it.